.github/workflows/ci.yml

```diff
@@ -12,10 +12,12 @@ jobs:
         java: [11]
         gradle: ['6.9.2']
         include:
-          - java: 15
+          - java: 17
             gradle: '7.3.3'
             properties: '-Pspock_version=2.0-groovy-3.0 -PgebVersion=5.0'
-
+          - java: 17
```

Comment:
I don't think it's worth running tests on both Gradle 7.3.3 and Gradle 7.4 - you could remove the older version. Otherwise great PR!

Reply:
Agreed, let's drop Gradle 7.3.3 👌